linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -226,10 +226,12 @@ async fn make_transfer_certificate_for_epoch<S>(
     }
     .with_simple_transfer(recipient, amount);
     let messages = match recipient {
-        Recipient::Account(account) => vec![direct_outgoing_message(
-            account.chain_id,
-            SystemMessage::Credit { account, amount },
-        )],
+        Recipient::Account(account) => {
```

Comment:
Do we need this change? (which will conflict with #1429, incidentally)

Reply:
(No, sorry, that must have been a formatting accident.)